exporter/signalfxexporter/internal/translation/converter.go

```diff
@@ -244,6 +247,18 @@ func (dpv *datapointValidator) isValidMetricName(name string) bool {
 	return true
 }
 
+func (dpv *datapointValidator) isValidNumberOfDimension(dp *sfxpb.DataPoint) bool {
+	if len(dp.Dimensions) > maxNumberOfDimensions {
+		dpv.logger.Warn("dropping datapoint",
```

Comment:
For high throughput metrics this will flood logs and I think debug would be preferable.

Reply:
true, I was just following the current standard in that file.
While I am at it, shall I change the others from warning to debug?